### DISCUSSION TITLE
Automatic security audit

### DISCUSSION THREAD

**Discussion by hpvd** [mysticaltech/terraform-hcloud-kube-hetzner#577] [Ideas]
Just found this really interesting looking project.

Since it's all about automation and security:
was there already a quick run with an independent tool, checking for best practice settings of a ready-installed Kubernetes cluster e.g. according to CIS Kubernetes benchmark (https://www.cisecurity.org/benchmark/kubernetes)?

If so it would be great to spread a word about it in readme (new checkmark!)
if not, maybe one should give it a try to bring up the last one or two small optimization possibilities...
e.g.
Checks whether Kubernetes is deployed according to security best practices as defined in the CIS Kubernetes Benchmark 
https://github.com/aquasecurity/kube-bench

**Comment by mysticaltech** [comment]
@hpvd Will do, thanks for the suggestion. Kube-bench is a good one, thanks for the reminder. We will perform a scan ASAP. 

And if you want to perform one, please don't hesitate and if you find something off, my email is in my profile.

Just FYI, for our future major version, we are also considering moving to RKE2 aka RKE Gov, and that one has many sec optimizations.
